Example/Example/AppDelegate.swift

```diff
@@ -1,7 +1,7 @@
 //
 //  AppDelegate.swift
 //
-//  Copyright (c) 2018 Virtusize AB
+//  Copyright (c) 2018-20 Virtusize AB
```

Comment:
@akueisara Just seeing that but it should be Virtusize KK, AB was the Swedish entity.

Could you make a separate feature branch to modify that in every files please?

Reply:
ok, will do today.